### DISCUSSION TITLE
Curation report Nov 21-27

### DISCUSSION THREAD

**Discussion by mkutmon** [wikipathways/GPML-Repo#103] [Curation reports]
- updated 4 curated/featured collection tags which were outdated 
  -  not sure about the change in [WP4874](https://www.wikipathways.org/index.php/Pathway:WP4874) - I didn't see any actual changes and there was no description - major issues with publication references in this pathway
- Request deletion for [WP5301](https://www.wikipathways.org/index.php/Pathway:WP5301)
- 1 new pathway [WP5303](https://www.wikipathways.org/index.php/Pathway:WP5303)
- Changes by LipidMAPS (Matthew), Nhung and Egon

@susancoort, you are up next.

**Comment by danidi** [comment]
In WP4874, there is an actual change: RPS6 leads to "Ankosis" in the old version, in the new one it is "Anoikis".

**Reply by mkutmon** [reply]
Ah thanks! Then I will update the curation tag too. Thanks!
